### PR TITLE
Update email_reply_to.html

### PR DESCRIPTION
**Pull request by fidejoseph**
Adding an extra line to see if this can reduce the number of personal reply-to emails being submitted

### CODE REVIEW

File: app/templates/views/service-settings/email-reply-to/add.html [hunk 1]
```diff
@@ -15,16 +15,12 @@
...
-  <p>
-    Do not use your own email address for replies.
-  </p>
+
   {% call form_wrapper() %}
     {{ textbox(
       form.email_address,
-      width='2-3',
+      width='1-1',
+      hint='This should be a shared inbox managed by your team, not your own email address.',
```
Comment (quis) [inline app/templates/views/service-settings/email-reply-to/add.html]:
```suggestion
      hint='This should be a shared inbox managed by your team, not your own email address',
```

**Comment by quis** [comment]
Looks like this now:

image

image

File: app/templates/views/service-settings/email-reply-to/_verify-updates.html [hunk 2]
```diff
@@ -50,14 +46,11 @@ <h2 class='banner-title' data-module="track-error" data-error-type="reply-to-ema
     ) %}
       {{ textbox(
         form.email_address,
-        width='2-3',
-        safe_error_message=True
+        width='1-1',
+        safe_error_message=True,
+        hint='This should be a shared inbox managed by your team, not your own email address'
       ) }}
-      {% if form.is_default.data %}
```
Comment (CrystalPea) [inline app/templates/views/service-settings/email-reply-to/_verify-updates.html]: I don't understand this change...?

Comment (quis) [reply app/templates/views/service-settings/email-reply-to/_verify-updates.html]: We paired on working out what the change should have been in 944a7d3

**Pull request merged**